packages/e2e/test/artillery/wallet-restoration/queries.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Query randomized distinct addresses from db associated with users who are staking
```

Comment:
```suggestion
 * Query randomized distinct addresses from db with an associated stake key.
```
You'd need to join to a delegation record to make the original assertion. Also, let's be precise and avoid using the term "staking". Users register a stake key and delegate their stake :slightly_smiling_face:

Reply:
> You'd need to join to a delegation record to make the original assertion

Not sure I got this, should I change the query somehow? Currently the query uses delegation records and joins with another tables. @rhyslbw

Reply:
> Currently, the query uses delegation records

Oh shoot, sorry, I didn't see the join. You can revert that change, as the original name was valid.

Reply:
Reverted in https://github.com/input-output-hk/cardano-js-sdk/pull/528/commits/0de3a4e87486464ebb2ab565d864ac5306ff1c9f @rhyslbw